tests/rest-api.test.js

```diff
@@ -521,10 +521,21 @@ describe('Rest API interface', () => {
         });
       });
 
+      it('should return rejected promise if registered callback does not match service configuration', (done) => {
+        nock(url)
+          .get('/notification/callback')
+          .reply(200, response.badNotificationCallback);
+        service.checkNotificationCallback()
+          .catch((err) => {
+            expect(err.code).to.equal('incorrectIpAdress');
+            done();
+          });
+      });
+
       it('should return rejected promise with exception object if connection is not succesfull', (done) => {
         service.checkNotificationCallback()
           .catch((err) => {
-            expect(typeof err).to.equal('object');
```

Comment:
Patikrinimą kad `err` yra objektas galima (ir rekomenduotina) palikti. Arba dar konkrečiau tikrinti, jog tai Error klasė (https://stackoverflow.com/questions/30469261/checking-for-typeof-error-in-js).